src/transformers/modeling_bart.py

```diff
@@ -938,6 +938,7 @@ def get_output_embeddings(self):
 )
 class BartForConditionalGeneration(PretrainedBartModel):
     base_model_prefix = "model"
+    keys_to_ignore_at_load = [r"final_logits_bias"]
```

Comment:
+ ['encoder.version, decoder.version']
```suggestion
    keys_to_ignore_at_load = [r"final_logits_bias", r"encoder.version", r"decoder.version"]
```

all bart variants should ignore those keys.

Reply:
would be good to check `MarianMTModel` which uses the `final_logits_bias` and runs pretty quickly.
```bash
RUN_SLOW=1 pytest --tb=short ./tests/test_modeling_marian.py -ra
```

I think (if I understand the change correctly) that a less confusing name would be: `keys_that_can_be_missing_without_concern`. 
When I started reading this I should you were removing them from state_dict if they were present, rather than manipulating the missing keys after the warning.

Reply:
The . will need a \ (since . is special in regex) so not commit the suggestion directly.